.github/workflows/security.yml

```diff
@@ -0,0 +1,94 @@
+name: Scan container images
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '0 4 * * 1'  # Every Monday at 9:30 IST
+
+env:
+  REGISTRY: docker.io
+  IMAGE_NAME: paralusio/paralus
+
+jobs:
+
+  build-scan-paralus:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Go
```

Comment:
This step can be removed because we no longer running go test.

Reply:
removed